docs/root/configuration/listeners/runtime.rst

```diff
@@ -6,3 +6,6 @@ The following runtime settings are supported:
 
 envoy.resource_limits.listener.<name of listener>.connection_limit
     Sets a limit on the number of active connections to the specified listener.
+
+envoy.extensions.filters.network.connection_limit.v3.ConnectionLimit
+    Connection limiting :ref:`architecture overview <arch_overview_connection_limit>`
```

Comment:
I would remove this from here. It's a runtime that is only applicable to the filter, and it's documented there.

Reply:
Removed.